scripts/utils/printing_tools.js

```diff
@@ -0,0 +1,55 @@
+const BN = require("bn.js")
+const bnOne = new BN(1)
+const bnTwo = new BN(2)
+const bnTen = new BN(10)
+const bn256 = new BN(256)
+const bnMaxUint = bnTwo.pow(bn256).sub(bnOne)
+
+/**
+ * A generalized version of "toWei" for tokens with an arbitrary amount of decimals.
+ * If the decimal representation has more decimals than the maximum amount possible, then the extra decimals are truncated.
+ * @param {string} amount User-friendly representation for the amount of some ERC20 token
+ * @param {integer} decimals Maximum number of decimals of the token
+ * @return {string} decimal representation of the number of token units corresponding to the input amount
+ */
+const fromUserToMachineReadable = function (amount, decimals) {
```

Comment:
Maybe the function names like `convertToNumberWithDecimalsDigits`, 'toGeneralizedWEI' or `convertToWeb3Representation` would be a better fit.